lib/routes/drivers/routes.js

```diff
@@ -0,0 +1,141 @@
+import { Sequelize } from 'sequelize';
+import Joi from '@hapi/joi';
+import {
+    addDriver,
+    updateDriver,
+    addDriverLocation,
+    acceptBooking,
+    updateBookingAttempts
+} from 'daos/driverDao';
+import {
+    emailSchema,
+    idAllowedSchema,
+    stringSchema
+} from 'utils/validationUtils';
+
+module.exports = [
+    {
+        method: 'POST',
+        path: '/add',
+        options: {
+            description: 'register new driver.',
+            notes: 'POST drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: {
+                payload: Joi.object({
+                    firstName: stringSchema,
+                    lastName: stringSchema,
+                    mobileNo: stringSchema,
+                    email: emailSchema,
+                    gender: Joi.string().valid('male', 'female', 'other'),
+                    cabTypeId: idAllowedSchema,
+                    cabNumber: stringSchema
+                })
+            }
+        },
+        handler: async (request, h) => {
+            try {
+                let driver = await addDriver(request.payload);
+                return h.response(driver).code(200);
+            } catch (error) {
+                return h.response(error.message).code(400);
+            }
+        }
+    },
+    {
+        method: 'PUT',
+        path: '/profile/update',
+        options: {
+            description: 'update driver profile.',
+            notes: 'PUT drivers API',
+            tags: ['api', 'drivers'],
+            cors: true,
+            auth: false,
+            validate: {
+                payload: Joi.object({
+                    id: idAllowedSchema,
+                    firstName: stringSchema,
+                    lastName: stringSchema,
+                    mobileNo: stringSchema,
+                    email: emailSchema,
+                    gender: Joi.string().valid('male', 'female', 'other'),
+                    cabTypeId: idAllowedSchema,
+                    cabNumber: stringSchema
+                })
+            }
+        },
+        handler: async (request, h) => {
+            try {
+                await updateDriver(request.payload);
+                return h
+                    .response('Driver profile updated successfully.')
+                    .code(200);
+            } catch (error) {
+                return h.response(error.message).code(400);
+            }
+        }
+    },
+    {
+        method: 'POST',
+        path: '/add_location',
```

Comment:
Let's not have snake cased routes please.

Reply:
Done.